Client/Frontend/Browser/Onboarding/OnboardingPrivacyConsentView.swift

```diff
@@ -0,0 +1,93 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import Shared
+import BraveShared
+import pop
+import SnapKit
+
+extension OnboardingPrivacyConsentViewController {
+    
+    class View: UIView {
+        
+        let yesConsentButton = CommonViews.primaryButton(text: Strings.OBPrivacyConsentYesButton).then {
+            $0.accessibilityIdentifier = "OnboardingPrivacyConsentViewController.YesButton"
+            $0.titleLabel?.adjustsFontSizeToFitWidth = true
+        }
+        
+        let noConsentButton = CommonViews.secondaryButton(text: Strings.OBPrivacyConsentNoButton).then {
+            $0.accessibilityIdentifier = "OnboardingPrivacyConsentViewController.NoButton"
+            $0.titleLabel?.adjustsFontSizeToFitWidth = true
+        }
+        
+        private let mainStackView = UIStackView().then {
+            $0.axis = .vertical
+            $0.distribution = .equalSpacing
+        }
+        
+        private let braveLogo = UIImageView(image: #imageLiteral(resourceName: "browser_lock_popup")).then {
+            $0.contentMode = .scaleAspectFit
+        }
+        
+        private let titleLabel = CommonViews.primaryText(Strings.OBPrivacyConsentTitle).then {
+            $0.font = .systemFont(ofSize: 20, weight: .semibold)
+            $0.numberOfLines = 0
+        }
+        
+        private let copyFromClipboardLabel = UILabel().then {
+            $0.text = "Allow Brave to check my clipboard for a matching referral code"
```

Comment:
@iccub - Shouldn't this be in BraveStrings.swift?

Reply:
this view is not used anymore, will delete